sync/src/synchronizer/mod.rs

```diff
@@ -675,11 +675,11 @@ mod tests {
         let num = 200;
```

Comment:
let num = 200u128;

Reply:
this doesn't help, 
in those test will also take the `num` as `BlockNumber`,  cast can't be avoided.
```
fn insert_block(
        chain_controller: &ChainController,
        shared: &Shared,
        nonce: u128,
        number: BlockNumber,
) 
```